ChangeLog

```diff
@@ -84,6 +84,10 @@ Release date: tba
       emitted when item deletion is tried on an object which doesn't
       have this ability. Closes issue #592.
 
+    * Added a new refactory message, 'consider-merging-isinstance', which is
+      emitted when the built-in method `isinstance` is used many times with
```

Comment:
``is used to check the type of the same object, with different calls, in the same chained boolean operation``.